rotkehlchen/assets/asset.py

```diff
@@ -236,6 +238,11 @@ def deserialize_from_db(cls, entry: UnderlyingTokenDBTuple) -> 'UnderlyingToken'
     strethaddress_to_identifier('0xbC396689893D065F41bc2C6EcbeE5e0085233447'): 'PERP',
     strethaddress_to_identifier('0xD417144312DbF50465b1C641d016962017Ef6240'): 'CQT',
     strethaddress_to_identifier('0xF5D669627376EBd411E34b98F19C868c8ABA5ADA'): 'AXS',
+    'WBTC': 'WBTC',
+    strethaddress_to_identifier('0x491604c0FDF08347Dd1fa4Ee062a822A5DD06B5D'): 'CTSI',
+    strethaddress_to_identifier('0xBBbbCA6A901c926F240b89EacB641d8Aec7AEafD'): 'LRC',
+    'KAR': 'KAR',
```

Comment:
why is this needed? Do we have multiple ones where the symbol is KAR? I mean it's not wrong to be explicit in case we ever get another token with symbol KAR, but this is not what we have been doing so far.

Reply:
For kraken we expect all of the assets to be in that list https://github.com/rotki/rotki/blob/develop/rotkehlchen/tests/exchanges/test_kraken.py#L55

Reply:
oh it was kraken. Sorry I missed it.

Reply:
It's due to the way github shows the diff

![Uploading 2021-08-03_13-04.png…]()